docs/_sass/footer.scss

```diff
@@ -1,6 +1,15 @@
 .container {
   text-align: center;
   display: flex;
+  flex-direction: column;
```

Comment:
@NikhilSharma03 I just checked using the inspect tool, this css change seems to be causing the increase in size of header. Header is also using the same `.container` class

Reply:
Thank you. I will fix it right away

Reply:
Looks good now, nice work